Apps/WebClient/src/NewClientApp/src/components/site/FeedbackComponent.vue

```diff
@@ -0,0 +1,272 @@
+);
+const isInvalid = computed<boolean>(
+    () => !v$.value.$dirty || (v$.value.$dirty && v$.value.$invalid)
+);
+const resultTitle = computed<string>(() => {
+    if (hasSubmitted.value) {
+        return isSuccess.value ? "Received" : "Sorry";
+    }
+
+    return "";
+});
+const resultDescription = computed<string>(() => {
+    if (hasSubmitted.value) {
+        return isSuccess.value
+            ? "Your message has been sent successfully!"
+            : "Your message could not be sent out!";
+    }
+
+    return "";
+});
+
+function showDialog(): void {
+    visible.value = true;
+}
+
+function hideDialog(): void {
+    visible.value = false;
+}
+
+function onSubmit(): void {
+    isLoading.value = true;
+    if (isInvalid.value) {
+        v$.value.$touch();
+        isLoading.value = false;
+        return;
+    }
+
+    userFeedbackService
+        .submitFeedback(user.value.hdid, {
+            comment: comment.value,
+        })
+        .then((result) => {
+            isSuccess.value = result;
+        })
+        .catch((err: ResultError) => {
+            if (err.statusCode === 429) {
+                errorStore.setTooManyRequestsError("page");
+            }
+            isSuccess.value = false;
+        })
+        .finally(() => {
+            hasSubmitted.value = true;
+            isLoading.value = false;
+        });
+}
+
+const commentErrors = computed(() =>
+    v$.value.comment.$errors.map((e) =>
+        typeof e.$message === "string" ? e.$message : e.$message.value
+    )
+);
+
+const isSuccessWithoutEmail = computed<boolean>(
+    () => isSuccess.value && !hasEmail.value
+);
+
+const isSuccessWithEmail = computed<boolean>(
+    () => isSuccess.value && hasEmail.value
+);
+
+const hasFailed = computed(() => !isSuccess.value && hasSubmitted.value);
+
+function resetFeedback(): void {
+    visible.value = false;
+    // begin closing before changing state
+    setTimeout(() => {
+        hasSubmitted.value = false;
+        isSuccess.value = false;
+        comment.value = "";
+        v$.value.$reset();
+    }, 500);
+}
+</script>
+
+<template>
+    <v-dialog
+        id="feedback-container"
+        v-model="visible"
+        data-testid="feedbackContainer"
+        persistent
+        no-click-animation
+    >
+        <v-form
+            v-model="isInvalid"
+            class="d-flex justify-center"
+            @submit.prevent="onSubmit"
+        >
+            <v-card max-width="700px">
+                <v-card-title class="px-0">
+                    <v-toolbar title="Feedback" density="compact" color="white">
+                        <HgIconButtonComponent
+                            data-testid="messageModalCloseButton"
+                            icon="close"
+                            @click="hideDialog"
+                        />
+                    </v-toolbar>
+                </v-card-title>
+                <v-card-text v-if="!hasSubmitted" class="text-body-1">
```

Comment:
```suggestion
                <v-card-text v-if="!hasSubmitted" class="pa-4 text-body-1">
```
Suggest adding this padding (which only affects the top) for consistency with `<MessageModalComponent />` and other dialogs.